tuning_configs.py

```diff
@@ -76,13 +75,13 @@
     "loss_computation": LOSS_COMPUTATION,
     "pseudo_bulk": PSEUDO_BULK,
     "encode_covariates": ENCODE_COVARIATES,
-    "encode_cont_covariates": ENCODE_CONT_COVARIATES,
     "mixup_penalty": MIXUP_PENALTY,
     "dispersion": DISPERSION,
     "gene_likelihood": GENE_LIKELIHOOD,
     "train_size": TRAIN_SIZE,
     "batch_size": BATCH_SIZE,
     "n_latent": LATENT_SIZE,
+    "n_cell_types": len(GROUPS[TRAINING_CELL_TYPE_GROUP]) - 1,
```

Comment:
what is the `GROUPS` variable?

Reply:
The one where we define all the granularities, in constants.py.
So GROUPS[TRAINING_CELL_TYPE_GROUP] is the chosen granularity, and len(GROUPS[TRAINING_CELL_TYPE_GROUP]) - 1 is the number of cell types of this granularity.